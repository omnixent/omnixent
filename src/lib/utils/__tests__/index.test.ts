import { normalizeServiceArgs } from '../';

test('Testing normalizeServiceArgs function', () => {
  expect(
    normalizeServiceArgs({ term: 'javascript', country: 'it', language: 'de' }),
  ).toMatchSnapshot();
  expect(
    normalizeServiceArgs({ term: 'blue jeans', country: 'us', language: 'en' }),
  ).toMatchSnapshot();
});
