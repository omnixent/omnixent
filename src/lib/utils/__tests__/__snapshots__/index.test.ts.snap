// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing normalizeServiceArgs function 1`] = `
Object {
  "c": "it",
  "hl": "it-DE",
  "l": "DE",
  "t": "javascript",
}
`;

exports[`Testing normalizeServiceArgs function 2`] = `
Object {
  "c": "us",
  "hl": "us-EN",
  "l": "EN",
  "t": "blue%20jeans",
}
`;
